easybuild/easyblocks/n/namd.py

```diff
@@ -43,10 +44,21 @@ def extra_options():
             # see http://charm.cs.illinois.edu/manuals/html/charm++/A.html
             'charm_arch': [None, "Charm++ target architecture", MANDATORY],
             'charm_opts': ['--with-production', "Charm++ build options", CUSTOM],
-            'namd_basearch': ['Linux-x86_64', "NAMD base target architecture (compiler family is appended", CUSTOM],
```

Comment:
actually, if someone out there has been using `namd_basearch` in easyconfigs, this will break them, wouldn't it have been safer to keep the extra option with 'None' as default and then set it in the easyblock only when it's not set in the easyconfig?

Reply:
Agreed, we shouldn't remove custom easyconfig parameters, that's not a backwards compatible change (especially since in EasyBuild 4.x we complain about unknown easyconfig parameters).

I'll open a PR to fix this.

Reply:
see #2123